### DISCUSSION TITLE
Slack Integration

### DISCUSSION THREAD

**Discussion by sonicaghi** [Kong/kong#7368] [Ideas and feature requests]
Kong send notifos into your slack account.

**Comment by subnetmarco** [comment]
What kind of notifications?

**Comment by ahmadnassri** [comment]
makes little sense to be a kong plugin ... rather a pager duty plugin, which in turn sends to slack ...

**Comment by tyiss** [comment]
Another use case for slack is, to get commands executed at slack channel in kong endpoint.
such as, update plugins configuration, get statistics, etc.

**Comment by sonicaghi** [comment]
yes that's what i meant too.. not just webhook/passive type of actions but also active commands via slack.

**Comment by nijikokun** [comment]
> Another use case for slack is, to get commands executed at slack channel in kong endpoint.
> such as, update plugins configuration, get statistics, etc.

Yes, this would be amazing.
- Commands for Administration API (statistics, configuration, version -- future: nodes)
- Commands for Adding / Managing API / Plugins

---

When clustering hits a potential plugin would be when nodes join / die, or monitoring of cassandra (but could be pager duty here as @ahmadnassri mentioned)
